.github/workflows/release.yaml

```diff
@@ -17,7 +22,15 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v1
         with:
-          node-version: 14
+          node-version: '${{ env.NODE }}'
+      - name: Set up npm cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-v${{ env.NODE }}-${{ hashFiles('package.json') }}-${{ hashFiles('package-lock.json') }}}
```

Comment:
I think there's an extra closing curly bracket here - also, is there a way to add the OS version to the cache key? (linux jut seems to be `linux` while I guess for different versions / distros we'd wanna have deps to be built ad-hoc

Reply:
The tray bracket is fixed. As for OS here, it's not needed since you only run this workflow on Linux.